docs/installation/centos.md

```diff
@@ -12,153 +12,185 @@ parent = "smn_linux"
+
+Finally, is it recommended that you fully update your system. Please keep in
+mind that CentOS 6 should be fully patched to fix any potential kernel bugs. Any
+reported kernel bugs may have already been fixed on the latest kernel packages 
+
+## Install
+
+You use the same installation procedure for all versions of CentOS,
+only the package you install differs. There are two packages to choose from:
+
+<table>
+  <tr>
+    <th>Version</th>
+    <th>Package name</th>
+  </tr>
+  <tr>
+    <td>6.5 and  higher</td>
+    <td>
+    <p>
+    <a href="https://get.docker.com/rpm/1.7.0/centos-6/RPMS/x86_64/docker-engine-1.7.0-1.el6.x86_64.rpm">
+    https://get.docker.com/rpm/1.7.0/centos-6/RPMS/x86_64/docker-engine-1.7.0-1.el6.x86_64.rpm</a>
+    <p>
+    <a href="https://get.docker.com/rpm/1.7.0/centos-6/SRPMS/docker-engine-1.7.0-1.el6.src.rpm">
+   https://get.docker.com/rpm/1.7.0/centos-6/SRPMS/docker-engine-1.7.0-1.el6.src.rpm</a>
+    <p>
+    </p>
+    </td>
+  </tr>
+  <tr>
+    <td>7.X</td>
+    <td>
+    <p>
+     <a href="https://get.docker.com/rpm/1.7.0/centos-7/RPMS/x86_64/docker-engine-1.7.0-1.el7.centos.x86_64.rpm">
+    https://get.docker.com/rpm/1.7.0/centos-7/RPMS/x86_64/docker-engine-1.7.0-1.el7.centos.x86_64.rpm</a>   
+    </p>
+    <p>
+     <a href="https://get.docker.com/rpm/1.7.0/centos-7/SRPMS/docker-engine-1.7.0-1.el7.centos.src.rpm">
+    https://get.docker.com/rpm/1.7.0/centos-7/SRPMS/docker-engine-1.7.0-1.el7.centos.src.rpm</a>   
+    </p>
+    </td>
+  </tr>
+</table>
 
-These instructions are likely work for other binary compatible EL6/EL7 distributions
-such as Scientific Linux, but they haven't been tested.
 
-Please note that due to the current Docker limitations, Docker is able to
-run only on the **64 bit** architecture.
+This procedure depicts an installation on version 6.5. If you are installing on
+7.X, substitute that package for your installation. 
 
-## Kernel support
+1. Log into your machine as a user with `sudo` or `root` privileges.
 
-Currently the CentOS project will only support Docker when running on kernels
-shipped by the distribution. There are kernel changes which will cause issues
-if one decides to step outside that box and run non-distribution kernel packages.
+2. Make sure your existing packages are up-to-date.
 
-To run Docker on [CentOS-6.5](http://www.centos.org) or later, you will need
-kernel version 2.6.32-431 or higher as this has specific kernel fixes to allow
-Docker to run.
+		$ sudo yum update
+		
+3. Download the Docker RPM to the current directory.
+		
+		$ curl -O -sSL https://get.docker.com/docker-engine-1.7.0-0.1.el6.x86_64.rpm
 
-## CentOS-7
+4. Use `yum` to install the package.
 
-### Installation
+		$ sudo yum localinstall --nogpgcheck docker-engine-1.7.0-0.1.el6.x86_64.rpm
 
-Docker is included by default in the CentOS-Extras repository. To install
-run the following command:
+5. Start the Docker daemon.
 
-    $ sudo yum install docker
+		$ sudo service docker start
 
```

Comment:
Thinking of SystemD; I don't think this works on CentOS/RHEL/Fedora 7 because of the switch to SystemD (only just realized this). So there are more differences between 6.x and 7.x then just "swapping" the package name :'(